### ISSUE TITLE
[Request] Links ft Simple Icons

### ISSUE THREAD

**Issue by luximus-hunter** [joelshepherd/tabliss#387]
Hiya,

I love the extension that you've made, but there is one feature that I'd really love to see:

The links currently have a set list of icons to choose from. I suggest adding support for [simple icons](https://github.com/simple-icons/simple-icons) to extend the amount of icons.

**Comment by wsdfhjxc** [comment]
I agree, the current choice of icons is quite limited, and lacks many concepts/brands.

Personally, I really like the aesthetics of the [Line Awesome](https://icons8.com/line-awesome) font, but Simple Icons is also a nice project.
